inngest/_internal/client_lib.py

```diff
@@ -321,6 +348,24 @@ def _get_sync(self, url: str) -> httpx.Response:
             ),
         )
 
+        if (
+            res.status_code
+            in (http.HTTPStatus.FORBIDDEN, http.HTTPStatus.UNAUTHORIZED)
+            and self._signing_key_fallback is not None
+        ):
+            # Retry with the fallback signing key
+            res = httpx.get(
+                url,
+                headers=net.create_headers(
+                    env=self._env,
+                    framework=None,
+                    server_kind=None,
+                    signing_key=self._signing_key,
```

Comment:
shouldn't this be the fallback key?

Reply:
Yea. I updated the PR to use a shared function to minimize mistakes like this